src/views/inputTest.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="w-1/3 px-10 py-5">
+    <h1 class="text-xl font-bold my-5">BaseInput component (email)</h1>
+    <ValidationObserver v-slot="{ valid }">
+      <form @submit.prevent="handleSubmit">
+        <base-input
+          v-model="email"
+          type="email"
+          name="email"
+          rules="email|required"
+          placeholder="you@example.com"
+          @input="clgEmail"
+          >Enter email
+        </base-input>
+        <button
+          type="submit"
+          class="
```

Comment:
just format the props on new lines, the css classes can be on the same line until it reaches the edge of the screen